mlxtend/feature_selection/exhaustive_feature_selector.py

```diff
@@ -111,24 +151,44 @@ class ExhaustiveFeatureSelector(BaseEstimator, MetaEstimatorMixin):
         An int, giving the exact number of total jobs that are spawned
         A string, giving an expression as a function
             of n_jobs, as in `2*n_jobs`
+
     clone_estimator : bool (default: True)
         Clones estimator if True; works with the original estimator instance
         if False. Set to False if the estimator doesn't
         implement scikit-learn's set_params and get_params methods.
         In addition, it is required to set cv=0, and n_jobs=1.
 
+    fixed_features : tuple (default: None)
+        If not `None`, the feature indices provided as a tuple will be
+        regarded as fixed by the feature selector. For example, if
+        `fixed_features=(1, 3, 7)`, the 2nd, 4th, and 8th feature are
+        guaranteed to be present in the solution. Note that if
+        `fixed_features` is not `None`, make sure that the number of
+        features to be selected is greater than `len(fixed_features)`.
+        In other words, ensure that `k_features > len(fixed_features)`.
```

Comment:
Technically, you are right regarding the greater-equal sign for `k_features > len(fixed_features)`. I mean, it would technically work, but then there is no feature selection happening, right? Or in other words, you already have the `fixed_features` you want to select so there is no point in using the `ExhaustiveFeatureSelector`.

For the case describing what happens if both `fixed_features` and `feature_groups` are specified. What do you think about the following?

> If both `fixed_features` and `feature_groups` are specified, ensure that each feature group contains the `fixed_features` selection. E.g., for a 3-feature set `fixed_features=[0, 1]` and `feature_groups=[[0, 1], [0, 1, 2]]` is valid; `fixed_features=[0, 1]` and `feature_groups=[[0, 1], [0, 2]]` is not valid.

Reply:
> but then there is no feature selection happening, right? Or in other words, you already have the `fixed_features` you want to select so there is no point in using the `ExhaustiveFeatureSelector`

RIGHT! I totally overlooked this! So, I am not going to change the description.


> For the case describing what happens if both `fixed_features` and `feature_groups` are specified. What do you think about the following?
> 
> > If both `fixed_features` and `feature_groups` are specified, ensure that each feature group contains the `fixed_features` selection. E.g., for a 3-feature set `fixed_features=[0, 1]` and `feature_groups=[[0, 1], [0, 1, 2]]` is valid; `fixed_features=[0, 1]` and `feature_groups=[[0, 1], [0, 2]]` is not valid.

So, I think you meant "`feature_groups=[[0, 1], [2]]` is valid;" .....   "`feature_groups=[[0], [1, 2]]` is not valid". Correct?